frontend/routes/viewer/components/issues/components/issueDetails/issueDetails.component.tsx

```diff
@@ -210,7 +219,11 @@ export class IssueDetails extends React.PureComponent<IProps, IState> {
 				onExpandChange={this.handleExpandChange}
 				renderCollapsable={this.renderDetailsForm}
 				renderNotCollapsable={() => this.renderLogList(comments && !!comments.length && !this.isNewIssue)}
-				handleHeaderClick={() => this.setCameraOnViewpoint({viewpoint: this.issueData.viewpoint})}
+				handleHeaderClick={() => {
+					if (this.issueData._id) { // if its a new issue it shouldnt go to the viewpoint
```

Comment:
```suggestion
					if (!this.isNewIssue) {
```
And then you can remove the comment